coverage.py

```diff
@@ -426,7 +497,11 @@ def __init__(self, _db, api, datasource, workset_size=10,
         metadata_replacement_policy = (
             metadata_replacement_policy or ReplacementPolicy.from_metadata_source()
         )
+        circulationdata_replacement_policy = (
```

Comment:
I see what's happening here and I'm okay with it for now, but I designed ReplacementPolicy to cover both the metadata and the circulation data. If we're passing in two objects, then they should be different objects -- a CirculationDataReplacementPolicy which covers links, rights, and formats, and a MetadataReplacementPolicy which covers links, identifiers, subjects, etc.

One change I do think needs to be made this branch is the default value for circulationdata_replacement_policy should be the same as metadata_replacement_policy, not the (potentially more permissive) ReplacementPolicy.from_license_source()